### DISCUSSION TITLE
plyer

### DISCUSSION THREAD

**Discussion by Gorkor227** [flet-dev/flet#3386] [Packages]
Plyer is a platform-independent Python API for accessing hardware features of various platforms (Android, iOS, macOS, Linux and Windows).

Plyer is managed by the [Kivy Team](https://kivy.org/about.html). It is suitable for use with Kivy apps, but can be used independently.

**Comment by baseplate-admin** [comment]
Hi,

I dont think `flet` team will be using `kivy-toolchain` in future

**Reply by FeodorFitsner** [reply]
Right, we will be further wrapping Flutter packages giving access to platform features. We already got geolocation, permissions handler, file picker, camera is coming just to name a few.

**Reply by Gorkor227** [reply]
Oh my god, thank you so much for replying to my discussion so quickly. This is the first time I am so close to a great engineer. I am a student from China and a believer of python. I really want to contribute to flet and try to expand it, but it is too difficult for me. I tried to make Flet-todo-app-Tutorials app, which is an open source example on github, but it is too old and can't run perfectly in the latest flet version. I fixed it, but I haven't open sourced it yet because I am not satisfied with my code structure. I have never made such a beautiful software. I recommended flet to several friends around me, and they thought it was incredible. Looking forward to the future of flet.
